software/base/src/main/java/brooklyn/entity/software/MachineLifecycleEffectorTasks.java

```diff
@@ -155,29 +155,42 @@ public void attachLifecycleEffectors(Entity entity) {
                 .build();
     }
 
+    /** @see {@link #newStartEffector()} */
+    public Effector<Void> newSuspendEffector() {
+        return Effectors.effector(Void.class, "suspend")
+                .description("Suspend the process/service represented by an entity")
+                .parameter(StopSoftwareParameters.STOP_PROCESS_MODE)
+                .parameter(StopSoftwareParameters.STOP_MACHINE_MODE)
+                .impl(newSuspendEffectorTask())
+                .build();
+    }
+
     /**
-     * Returns the {@link TaskFactory} which supplies the implementation for the start effector.
+     * Returns the {@link EffectorBody} which supplies the implementation for the start effector.
      * <p>
      * Calls {@link #start(Collection)} in this class.
      */
     public EffectorBody<Void> newStartEffectorTask() {
-        return new EffectorBody<Void>() {
-            @Override
-            public Void call(ConfigBag parameters) {
-                Collection<? extends Location> locations  = null;
+        return new StartEffectorBody();
```

Comment:
It will affect anybody who ran a 0.8.0-SNAPSHOT server with persistence since https://github.com/apache/incubator-brooklyn/pull/735 was closed, on the tenth of July, right? What do you think? Revert fcb1af0cbd7eb9799e48471e37934b046330baf3 and only name the new suspend effector? How many users are using the latest snapshots?